docs/examples/test/test_examples.py

```diff
@@ -48,6 +49,33 @@ def test_advanced(self, container_factory, rabbit_config):
         assert res.status_code == 200
         assert res.text == 'payload'
 
+    def test_expected_exception(self, container_factory, rabbit_config):
+
+        from examples.http_exceptions import Service
+
+        container = container_factory(Service, rabbit_config)
```

Comment:
i don't this (or indeed a bunch of other tests in this module) needs a `rabbit_config`